### PR TITLE
PE-7099: Staking modal updates

### PR DESCRIPTION
**Pull request by kunstmusik**
* Add expedited or standard withdrawal selections to initial unstake UI/UX
* Remove EAY from un-staking modal
* Add friction to staking modal (with review modal)
* Add friction to unstaking modal (with review modal)
* Change term unstake to withdraw

### CODE REVIEW

File: src/components/icons/circle_check.svg
Comment (atticusofsparta) [inline src/components/icons/circle_check.svg:1]: lucide has this one

File: src/components/icons/circle.svg
Comment (atticusofsparta) [inline src/components/icons/circle.svg:1]: lucide has this

File: src/components/icons/index.ts
Comment (atticusofsparta) [inline src/components/icons/index.ts:1]: we should make a ticket to get rid of these and use strictly icons from one place (ideally)

File: src/components/modals/ReviewStakeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  ticker,
+}: {
+  gateway: AoGatewayWithAddress;
+  amountToStake: number;
+  walletAddress: ArweaveTransactionID;
+  onClose: () => void;
+  onSuccess: () => void;
+  ticker: string;
+}) => {
+  const queryClient = useQueryClient();
```
Comment (atticusofsparta) [inline src/components/modals/ReviewStakeModal.tsx:32]: Question: whats the difference for us acquiring the current queryClient from the hook vs the constant?

File: src/components/icons/circle.svg
Comment (kunstmusik) [reply src/components/icons/circle.svg:1]: This SVG is from Lucide

File: src/components/icons/index.ts
Comment (kunstmusik) [reply src/components/icons/index.ts:1]: Yeah might be nice to remove the icons from Lucide in the app, add the lucide package, and use those instead. We'll probably still need this index for some custom icons (arconnect, ario.svg, etc.). Will take a bit of time to audit so a separate ticket seems best. (Added PE-7177 for this)

File: src/components/modals/ReviewStakeModal.tsx (hunk 1 above)
Comment (kunstmusik) [reply src/components/modals/ReviewStakeModal.tsx:32]: It's so the components can be reusable across projects and don't depend on a concrete instance.

**Review by atticusofsparta — APPROVED** [review]

**Pull request merged**